frameworks/Python/web2py/install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir -p $IROOT/.pip_cache
+
+export PIP_DOWNLOAD_CACHE=$IROOT/.pip_cache
+
+fw_depends python2
+
+export PY2_ROOT=$IROOT/py2
+export PY2_PIP=$PY2_ROOT/bin/pip
+
+$PY2_PIP install --install-option="--prefix=${PY2_ROOT}" -r $TROOT/requirements.txt
+
+cd $TROOT
+rm -fr web2py
+git clone --recursive https://github.com/web2py/web2py.git 
```

Comment:
Thanks! Is there any reason we should not pin this to a specific tag e.g. `R-2.10.3`? That would make the version used explicit, so anyone looking at the results in the future would be able to tell exactly what commit we were using